src/Security/PasswordValidator.php

```diff
@@ -112,8 +110,7 @@ public function characterStrength($minScore, $testNames = null)
     }
 
     /**
-     * @deprecated 4.5.0
-     * Check a number of previous passwords that the user has used, and don't let them change to that.
+     * @deprecated 4.5.0 Check a number of previous passwords that the user has used, and don't let them change to that.
```

Comment:
```suggestion
     * Check a number of previous passwords that the user has used, and don't let them change to that.
     * @deprecated 4.5.0 Use setHistoricCount() instead.
```
Update the notice message as well.

Reply:
Updated